web/sites/default/config/views.view.search_index.yml

```diff
@@ -9,6 +9,8 @@ dependencies:
     - field.storage.node.field_p_card_allowed
     - field.storage.node.field_type
     - search_api.index.full_index
+    - system.menu.main
+    - user.role.authenticated
```

Comment:
@kxwang , we should not limit the contacts search to authenticated. Anonymous users can only access the site if they are on the city network and we want employees on the network to be able to look up this information.

Reply:
Removed